apps/thunderhub/data/entrypoint.sh

```diff
@@ -3,4 +3,4 @@
 # Set password
 sed -i 's/$APP_PASSWORD/'${APP_PASSWORD}'/' /data/thubConfig.yaml
 
-exec npm start
+npm run start:prod
```

Comment:
This threw me for a loop. The entrypoint for thunderhub changed so `exec npm start` was actually the line causing the issue.

Reply:
This is not the correct change